### ISSUE TITLE
SoftBan

### ISSUE THREAD

**Issue by Lynomm** [finndev/PokeBuddy#685]
How do i know if my account is in softban and how do i disable it ?

(Getting 0 Exp / Always failed to catch pokemon / 0 items in pokestops)

**Comment by curtiola** [comment]
Quickspin can tell you if you're soft banned and can unban you as well, just be sure to set the coords to where you set pokebuddy to

**Comment by Lynomm** [comment]
Thanks a lot mate. Will try it now.

**Comment by Thorben24** [comment]
It is still working. 

Be sure you download it here: 

http://www.ownedcore.com/forums/pokemon-go/pokemon-go-hacks-cheats/565264-quickspin-pokemongo-softban-unbanner-quickly-unbans-your-softbanned-account.html

There are many fakes included malware available. If you are using google, make sure to disable 2 factor auth while you use Quickspin. (You can, and to be honest should activate it after).

**Comment by RoseCorpse** [comment]
just spin a pokestop 40 times and puff, softban is gone, easy.

**Comment by Thorben24** [comment]
@RoseCorpse Exactly thats what the quickspin tool is doin, just for the lazy ones :laughing:

**Comment by Lynomm** [comment]
You can't spin a pokestop for 40x using pokebuddy. Or can you?

**Comment by curtiola** [comment]
No, you can't lol quickspin is the way to go, but I think rose meant with the real pogo app

**Comment by Thorben24** [comment]
@Lynomm PokeBuddy usually did this automatically if it detect your account is softbanned. Somehow since the last pokebuddy update this isnt possible anymore.

**Comment by TFSho** [comment]
*sigh* Oh Softbans.
Was copying coords from Google, lost a minus symbol.
#othersideoftheworld
#goodmorningfromquiteliterallythemiddleofnowhere,China
#min4hourban
¯\_(ツ)_/¯
Welp, guess I'm going to bed

**Comment by p3monty** [comment]
I got softbanned instantly after launching a 2nd instance of pokebuddy. anyone else?

**Comment by Thorben24** [comment]
@p3monty Its an IP adress ban, not a softban. Restart your router or relog it so you will get a new ip adress.

**Comment by curtiola** [comment]
@bt has some interesting input on the subject in #669

**Comment by p3monty** [comment]
@Thorben24 hm thats strange. i thought ip ban is not seeing pokestops and other things and softban is not getting xp/pokemon. I see all pokestops and can try to catch pokemon but they always get away. Ofc in the meanwhile all is fine again but in my point of view it was a softban. Anyway thx for trying to help me. appreciated
